src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -33,4 +43,27 @@ public static boolean isNetworkAvailable(final Context context) {
         return connectivityManager.getActiveNetworkInfo() != null && 
             connectivityManager.getActiveNetworkInfo().isConnectedOrConnecting();
     }
+
+    public static void sendFeedEvent(Context context, String category) {
+        Log.d(TAG, "Logging feed event. Category is " + category);
+
+        getTracker(context).send(new HitBuilders.EventBuilder()
+                .setCategory(category)
+                .setAction("click")
+                .build());
+    }
+
+    private static Tracker getTracker(Context context) {
+        if (tracker == null) {
```

Comment:
Oh, that's a good idea actually. We could just have StartResult return the tracker already created. Then there's no need to have two. Ok, I'll make that change!